python/cudf/cudf/tests/test_parquet.py

```diff
@@ -629,15 +629,28 @@ def test_parquet_reader_spark_timestamps(datadir):
 def test_parquet_reader_spark_decimals(datadir):
     fname = datadir / "spark_decimal.parquet"
 
-    expect = pd.read_parquet(fname)
-    got = cudf.read_parquet(fname)
+    # expect = pd.read_parquet(fname)
+    with pytest.raises(
+        NotImplementedError,
+        match="Decimal type greater than Decimal64 is not yet supported",
+    ):
+        _ = cudf.read_parquet(fname)
 
     # Convert the decimal dtype from PyArrow to float64 for comparison to cuDF
     # This is because cuDF returns as float64 as it lacks an equivalent dtype
-    expect = expect.apply(pd.to_numeric)
+    # expect = expect.apply(pd.to_numeric)
 
     # np.testing.assert_allclose(expect, got)
-    assert_eq(expect, got)
+    # assert_eq(expect, got)
+
+
+def test_parquet_reader_decimal128_error_validation(datadir):
+    fname = datadir / "nested_decimal128_file.parquet"
```

Comment:
what is the schema of this file?

Reply:
This is the current schema:
```
>>> m.schema
<pyarrow._parquet.ParquetSchema object at 0x7f7a5dc52ec0>
required group field_id=-1 schema {
  optional group field_id=-1 a (List) {
    repeated group field_id=-1 list {
      optional fixed_len_byte_array(9) field_id=-1 item (Decimal(precision=20, scale=1));
    }
  }
}

```

Do you want the test to include some specific schema?